config.go

```diff
@@ -1,10 +1,39 @@
-package connectorname
-
-// Config contains shared config parameters, common to the source and
-// destination. If you don't need shared parameters you can entirely remove this
-// file.
-type Config struct {
-	// GlobalConfigParam is named global_config_param_name and needs to be
-	// provided by the user.
-	GlobalConfigParam string `json:"global_config_param_name" validate:"required"`
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+//go:generate paramgen -output=paramgen_dest.go DestinationConfig
+
+type DestinationConfig struct {
+	// The keyspace (similar to a database in a relational database system) that has the table.
+	Keyspace string `json:"keyspace" validate:"required"`
+	// The table name.
+	Table string `json:"table" validate:"required"`
+	// The host to access Cassandra.
+	Host string `json:"host" validate:"required"`
```

Comment:
I was actually thinking about this and wanted to add it after building the basics and getting the inserts working, but in general, the host here should be a list of hosts, cuz `gocql.NewCluster` takes a list of hosts, so you can provide all the nodes hosts here (but the port for all these hosts has to be the same), if one of the nodes has a different port then it would need a separate session, which would make things more complicated.
So what I'm thinking about is giving the user the freedom to add all the nodes hosts as a list in configs (comma-separated string), but the user is obligated to have the same port number for all of them, WDYT?

Reply:
> So what I'm thinking about is giving the user the freedom to add all the nodes hosts as a list in configs (comma-separated string), but the user is obligated to have the same port number for all of them, WDYT?

I think this makes sense. :+1: I don't think it necessarily needs to be part of this PR.